### ISSUE TITLE
Heading levels are not identified

### ISSUE THREAD

**Issue by paulettefrancis** [cds-snc/covid-shield-mobile#176]
Using VoiceOver screen reader

Headings should be marked up using levels such as Heading level 1, level 2, etc. on all pages.

The screen reader only reads "Heading" and not the level. Only the first heading is read.

![Screenshot 2020-06-25 at 4 06 56 PM](https://user-images.githubusercontent.com/22638158/85791004-27de0980-b6ff-11ea-8cbb-7ae314f9dbe3.png)

iOS XR version 13.5.1

**Comment by qanitah** [comment]
This is also for Android with TalkBack. It announces only "heading heading" for the top heading on the page. Other headings on the pages are not marked as headings (only a few have more headings).

Tested with Samsung Android version 8.1.0

**Comment by sarasuri** [comment]
Scenario

Samsung Galaxy J3 Prime 
Android version 7.0

With Talkback on, it announces the top level heading (h1) as "heading heading" on the screens not the level of the heading.

Expected result
It should identify the heading levels.

WCAG
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Comment by nickmly** [comment]
Looks like this issue relates to React Native not supporting heading levels:
https://reactnative.dev/docs/accessibility#accessibilityrole-android-ios

Unfortunately, they only accept an `accessibilityRole` of `header` which will be treated as an `h1`. 

As for TalkBack reading it as "heading heading", this issue also occurs on the upstream repo, so I am unsure if this is being looked at, or is possible to fix due to React Native.
